pkg/operator/deploy/deploy.go

```diff
@@ -52,18 +53,18 @@ type CRD struct {
 // platform to deploy on, whether or not to delete the metering CRDs,
 // or namespace during an install, the location to the manifests dir, etc.
 type Config struct {
-	Namespace                string
-	Platform                 string
-	DeployManifestsDirectory string
-	MeteringCR               string
 	SkipMeteringDeployment   bool
 	DeleteCRDs               bool
 	DeleteCRB                bool
 	DeleteNamespace          bool
 	DeletePVCs               bool
 	DeleteAll                bool
+	Namespace                string
+	Platform                 string
+	DeployManifestsDirectory string
```

Comment:
I feel like this should be removed from the `Config` struct and all files should be moved to being read outside of the Deployer code

Reply:
I agree, like managing a `MeteringConfig` structure directly, we could also handle building up the YAML manifests outside of the deployer methods and instead manage the Metering CR structures. I'd prefer to do this in a follow-up PR as that's a bigger change.

Reply:
Okay. That's fine.